tests/console/commands/env/test_activate.py

```diff
@@ -79,6 +79,7 @@ def test_env_activate_prints_correct_script_on_windows(
 
 
 @pytest.mark.parametrize("verbosity", ["", "-v", "-vv", "-vvv"])
+@pytest.mark.skipif(not WINDOWS, reason="Only Windows shells")
```

Comment:
I suppose there is no shell activator script that is available on all Linux systems? I assume that the test fails because `_get_activate_command` checks if an `activate.ps1` file exists, which does not in your case? (Interesting enough it seems to exist on Ubuntu - and it also works on macOS in our CI.)

I would prefer not restricting the test to Windows. Maybe, we can just patch `_get_activate_command` to just return an arbitrary string?

Reply:
So the files are present even on Fedora:
~~~~
(Pdb) os.listdir("/tmp/pytest-of-root/pytest-26/test_no_additional_output_in_v0/venv/bin")
['python', 'python3', 'python3.13', 'activate', 'activate.csh', 'activate.fish', 'activate.nu', 'activate.ps1', 'activate_this.py']
~~~~
I think the test is missing `mocker.patch("poetry.utils.env.EnvManager.get", return_value=tmp_venv)`, with this line the test passes.
Although I am not sure how come it works on Ubuntu a MacOS.

Reply:
That might also be a solution. I think it works because `MockEnv` works for our test setup, but obviously not for your setup. Maybe, it is not a matter of OS, but you do something different than our CI (or other contributors). On my system, it looks into the directory of Poetry's own venv, which also has these files.

Nevertheless, if `tmp_venv` works for you and our CI that is fine. It is a bit more expensive than `MockEnv`, but also more realistic.